tests/integration_tests/build/test_coverage.py

```diff
@@ -17,9 +17,9 @@
 # Checkout the cpuid crate. In the future other
 # differences may appear.
 if utils.is_io_uring_supported():
-    COVERAGE_DICT = {"Intel": 78.20, "AMD": 79.09, "ARM": 82.19}
+    COVERAGE_DICT = {"Intel": 78.08, "AMD": 78.92, "ARM": 81.88}
```

Comment:
is it clear to us what causes the drop and how to recover?

Reply:
I have added tests to smooth these numbers out. The ARM thresholds still have a slight drop, some of this is related to error handling that is in place in case of system errors that are hard to replicate in a unit test environment. The drop is very small though, for x86, coverage has been increased.